docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,404 @@
+
+Via the CLI:
+
+The alias set earlier can be confirmed using: `aztec-wallet get-alias accounts:main`, this is specified here in `--from main`.
+
+```bash
+SPONSORED_FPC_ADDRESS=<address>
+aztec-wallet register-contract SPONSORED_FPC_ADDRESS SponsoredFPC --from main # Need to specify account that wishes to register the contract
+aztec-wallet deploy-account --from main --payment method=fpc-sponsored,fpc=contracts:sponsoredfpc
+```
+
+The equivalent using aztec.js - get sponsored fpc address (helper functions [here](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/aztec/src/sandbox/sponsored_fpc.ts
+)) and use payment method:
+
+```javascript
+import { getSponsoredFPCInstance } from "../src/utils/sponsored_fpc.js"; // helper functions linked above
+import { SponsoredFPCContract } from "@aztec/noir-contracts.js/SponsoredFPC";
+import { SponsoredFeePaymentMethod } from "@aztec/aztec.js/fee/testing";
+  //... building upon the previous section
+  const sponseredFPC = await getSponsoredFPCInstance(); // get address of pre-deployed Sponsored FPC contract
+  await pxe.registerContract({instance: sponseredFPC, artifact: SponsoredFPCContract.artifact}); // register the Sponsord FPC contract class with the pxe
+  const sponsoredPaymentMethod = new SponsoredFeePaymentMethod(sponseredFPC.address); // create payment method
+  let tx = await schnorrAccount.deploy({ fee: { paymentMethod: sponsoredPaymentMethod }}).wait();
+  let wallet = await schnorrAccount.getWallet(); let address = wallet.getAddress();
+
+```
+
+:::note Payment: Sponsored Fee Paying Contract
+Options for payment via the sponsored fpc that can be used in multiple commands:
+- CLI: `--payment method=fpc-sponsored,fpc=$SPONSORED_FPC_ADDRESS`
+- .js: `{ fee: { paymentMethod: new SponsoredFeePaymentMethod(sponseredFPCAddress) }}`
+:::
+
+**Congratulations! You have successfully created an account on Aztec!**
+
+This contract now exists in the sandbox network, or on testnet if you specified a node url.
+
+### Fee Juice
+
+Apart from the FPC payment methods, the default method for paying for transactions is via fee juice direct from the sender of the tx.
+When specifying fee juice, and only in the special case of deploying an account, a different funded account may be specified to pay for the deployment transaction of an unfunded account.
+Another way for an accounts deployment to be funded is via claiming bridged fee juice (in a later section).
+
+#### Sandbox pre-funded test accounts
+
+The sandbox starts with 3 test accounts, providing another way to bootstrap initial testing with accounts. To add these to the PXE...
+
+For the CLI:
+```bash
+aztec-wallet import-test-accounts
+```
+
+Confirm with: `aztec-wallet get-alias`, to see all aliases
+
+The equivalent using aztec.js - get the test accounts
+```javascript
+  import { getInitialTestAccountsWallets } from "@aztec/accounts/testing";
+  //...
+    testWallets = await getInitialTestAccountsWallets(pxe);
```

Comment:
And this only works when running the sandbox with the PXE attached. You can't use it when running an independent PXE.

Reply:
This is in the section `#### Sandbox pre-funded test accounts`, but are you saying that if they're running the sandbox and pxe separately, they can't do this?